pyuia/pageobject.py

```diff
@@ -58,7 +56,14 @@ def _page_object(self, page_class, wait_for_loaded=True):
         if wait_for_loaded: page.wait_for_page_loaded()
         return page
 
-    def wait_for_page_loaded(self, timeout_warn=_WARN_TIMEOUT, timeout=_WAIT_TIMEOUT):
+    def wait_for_page_loaded(self, timeout_warn=None, timeout=None):
+        """Defalt timeout_warn is assigned by _PAGE_WARN_TIMEOUT
```

Comment:
It is recommended to revise your docstrings to conform to [Google Python Style Guide](http://google-styleguide.googlecode.com/svn/trunk/pyguide.html?showone=Comments#Comments).